core/store/src/db.rs

```diff
@@ -670,7 +670,7 @@ impl InstanceCounter {
         let (lock, cvar) = &*ROCKSDB_INSTANCES_COUNTER;
         let mut num_instances = lock.lock().unwrap();
         *num_instances += 1;
-        debug!(target: "db", "Created a new RocksDB instance. Current #instances: {}", *num_instances);
+        info!(target: "db", "Created a new RocksDB instance. Current #instances: {}", *num_instances);
```

Comment:
Should we also change this and the other log line to:
```rust
info!(target: "db", num_instances=%num_instances, "Created a new RocksDB instance." Current #instances: {}");
```
We are kinda moving towards more structured logging.

Reply:
Thanks.